package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@eqworks/lumen-table",
-  "version": "1.2.0",
+  "version": "1.2.1",
```

Comment:
would this be considered a fix/patch or new feature/behaviour (that is functionaly backward compatible)?

Reply:
it's just style so the table doesn't stretch. Doesn't change the implementation of the `<Table>` comp but changes the look of it if the string inside the cell is too long and doesn't have white spaces.

Reply:
I expected that table would do that already